### PR TITLE
get additional mesos and cassandra info on failure

### PR DESCRIPTION
**Pull request by falseperfection**
Need to update TeamCity configuration to take `*_slaves.json` and `*_state.json` as artifacts.

### CODE REVIEW

**Comment by benclarkwood** [comment]
@falseperfection do you mean "cluster" in the ticket title?

**Comment by falseperfection** [comment]
No, I meant cassandra. The ticket has these commands in it:
- `dcos cassandra plan list`
- `dcos cassandra plan show <plan-name>`

File: testing/sdk_utils.py [hunk 1]
```diff
@@ -37,11 +37,20 @@ def get_zk_path(service_name):
     return "test__integration__" + service_name
 
 
+def is_test_failure(request):
```
Comment (mellenburg) [inline testing/sdk_utils.py]: This feels kinda awkward here, but I get why you need it here. Could you please add a doc-string explaining that this is for ingesting pytest plugin requests, not the beloved requests lib used throughout this repo

**Comment by benclarkwood** [comment]
There were test failures for C*, but I don't see mesos state?

**Comment by falseperfection** [comment]
Sure enough. I need to add those files as artifacts (they're .json not .log). Also, it looks like I'm not saving state per-test. I'll update both things.

**Pull request merged**